### DISCUSSION TITLE
Submod support

### DISCUSSION THREAD

**Discussion by SoniEx2** [modrinth/.github#88] [Feature requests]
**Is your feature request related to a problem? Please describe.**

Sometimes mods come in modules. Like old-old buildcraft-common buildcraft-transport buildcraft-builders buildcraft-scam buildcraft-energy etc.

**Describe the solution you'd like**

There should be a way to make mod packages, mod groups, or submods (name is bikesheddable). They'd only bring up one item in search but with all the relevant submods.

**Describe alternatives you've considered**

The alternative of doing nothing is suboptimal because it creates a discovery problem (can't find the extra mods). The alternative of selling mod authors on putting everything into a single mod is also suboptimal because that's not how they wanna do it in the first place.

**Additional context**

N/A

**Comment by woodiertexas** [comment]
Didn't enderIO have a modular setup as well? (As well as a monolithic setup)

**Comment by SoniEx2** [comment]
a lot of mods did. curseforge kinda killed them. we could bring them back.

even fabric (fabric api) is modular.

**Comment by woodiertexas** [comment]
How did curseforge kill modular mods?

**Comment by SoniEx2** [comment]
we remember them dying pretty much somewhere around the curseforge era.

**Comment by Minenash** [comment]
One of the features that is planned is Project lists. I think having the base part of the mod listed with the modules being unlisted, but in a project list linked by the base mod's page would be a good option
